packages/frontend/src/redux/slices/account/createAccountThunks.js

```diff
@@ -116,6 +119,7 @@ export const createNewAccount = createAsyncThunk(
 
         await wallet.saveAndMakeAccountActive(accountId);
         await dispatch(addLocalKeyAndFinishSetup({ accountId, recoveryMethod, publicKey, previousAccountId })).unwrap();
+        dispatch(checkAndHideLedgerModal());
```

Comment:
@marcinbodnar please correct me if I'm wrong but I don't think we need an explicit `checkAndHideLedgerModal` if we're dispatching a redux action?

Reply:
You're right, ledger models are closed automatically. I see that @gutsyphilip already changed it.